content/metadata-stats/metadata/rest.textile

```diff
@@ -36,7 +36,7 @@ The following parameters are supported:
 
 - limit := _100_ optionally specifies the maximum number of results to return. The maximum is 1000.<br>__Type: @integer@__
 - prefix := optionally limits the query to only those channels whose name starts with the given prefix<br>__Type: @string@__
-- by := _value_ optionally specifies what to return. Use @by=id@ to return just channel names. Use @by=value@ to return "@ChannelDetails@":/api/realtime-sdk/channel-metadata#channel-details. The default has changed: it used to be @by=value@, but for for protocol v3 or newer (the @version@ parameter passed to @rest#request@ is ≥ 3), it is @by=id@. **You are strongly recommended to use @by=id@ unless you really need the occupancy for every channel**. For one thing it will be much faster; for another, making very regular @by=value@ enumeration queries can prevent channels from closing down from inactivity.
+- by := _id (v3) or value (v2)_ optionally specifies what to return. Use @by=id@ to return just channel names. Use @by=value@ to return "@ChannelDetails@":/api/realtime-sdk/channel-metadata#channel-details. Using @by=id@ is recommended unless occupancy data is needed for every channel. @by=value@ queries can keep channels active but may return results more slowly.
```

Comment:
> can keep channels active but may return results more slowly.

I don't understand the purpose of this change. Those are both bad things, joining them with "but" makes it sound sarcastic -- "It tastes terrible, but at least the portions are tiny".

I also not sure why you're trying to tone down the recommendation. There's no "may" about it, it _will_ be much slower and generally worse, and we want to discourage people from using it unless they absolutely have to.